core/cosmosdb/cache-invalidator/src/main/scala/org/apache/openwhisk/core/database/cosmosdb/cache/ChangeFeedConsumer.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.openwhisk.core.database.cosmosdb.cache
+
+import java.util
+
+import akka.Done
+import akka.event.slf4j.SLF4JLogging
```

Comment:
Any reason this shouldn't use the same Logging/AkkaLogging classes from common/scala? I guess this applies to all the other classes in cacheinvalidator.

Reply:
In most cases (except the one to log the progress) the logs were mostly for lifecycle related messages (startup/shutdown) and the CosmosDB SDK also logs using slf4j directly so either of those logging should be fine.

Switched to AkkaLogging now for consistency